wagtailio/project_styleguide/templates/patterns/pages/search_results/search_results.html

```diff
@@ -12,7 +12,10 @@ <h1 class="hero__heading">Search results</h1>
                         <label for="search" class="u-sr-only">Search</label>
                         <input id="search" type="search" placeholder="Enter your search term" class="form__input"
```

Comment:
```suggestion
                        <input name="query" id="search" type="search" placeholder="Enter your search term" class="form__input"
```

Reply:
Thanks! This fixed the issue with the search form not working! :rocket:

Reply:
fixed in 21a32065